ragna/assistants/_api.py

```diff
@@ -16,22 +17,28 @@ def requirements(cls) -> list[Requirement]:
         return [EnvVarRequirement(cls._API_KEY_ENV_VAR)]
 
     def __init__(self) -> None:
-        self._client = httpx.AsyncClient(
+        self._api_key = os.environ[self._API_KEY_ENV_VAR]
+
+        kwargs: dict[str, Any] = dict(
             headers={"User-Agent": f"{ragna.__version__}/{self}"},
             timeout=60,
         )
-        self._api_key = os.environ[self._API_KEY_ENV_VAR]
+        self._sync_client = httpx.Client(**kwargs)
+        self._async_client = httpx.AsyncClient(**kwargs)
```

Comment:
Since the Google assistants need a sync client, I've renamed the `._client` attribute to `._async_client` to better reflect its nature.